src/components/Popover/components/PopoverOverlay/PopoverOverlay.tsx

```diff
@@ -51,7 +51,7 @@ export interface PopoverOverlayProps {
   fixed?: boolean;
   hideOnPrint?: boolean;
   onClose(source: PopoverCloseSource): void;
-  colorScheme?: InversableColorScheme;
+  colorScheme?: NonNullable<ThemeProviderProps['theme']>['colorScheme'];
```

Comment:
Curious what `NonNullable` is doing here if the prop is optional?

Reply:
Note where the Generic ends (spacing added for emphasis) 

```
NonNullable<ThemeProviderProps['theme']>    ['colorScheme']
```

`ThemeProviderProps['theme']` is  `ThemeConfig | undefined`. The NonNullable knocks off the undefined bit of that so we have an object and thus can access the type of `ThemeConfig['colorScheme']` (which is itself potentially undefined)